### PR TITLE
test_id:713: make failure slightly more legible

### PR DESCRIPTION
**Pull request by dankenigsberg**
Let the failure text state the numbers which failed comparison.

```release-note
NONE
```

### CODE REVIEW

File: tests/vmi_headless_test.go [hunk 1]
```diff
@@ -115,7 +116,9 @@ var _ = Describe("[rfe_id:609]VMIheadless", func() {
 				memDiff := normalComputeContainer.Resources.Requests.Memory()
 				memDiff.Sub(*computeContainer.Resources.Requests.Memory())
 
-				Expect(memDiff.ScaledValue(resource.Mega) > 15).To(BeTrue(), "memory difference between headless and normal should be roughly 16M")
+				Expect(memDiff.ScaledValue(resource.Mega) > 15).To(BeTrue(),
+					fmt.Sprintf("memory difference between headless (%s) and normal (%s) should be roughly 16M",
+						computeContainer.Resources.Requests.Memory(), normalComputeContainer.Resources.Requests.Memory()))
```
Comment (ezrasilvera) [inline tests/vmi_headless_test.go:121]: nit: Why not go all the way and print the actual difference as well ?

Comment (dankenigsberg) [reply tests/vmi_headless_test.go:121]: To me, the diff is not helpful for debug. I need to understand which of the two is too big or too little.

Comment (dankenigsberg) [reply tests/vmi_headless_test.go:121]: @ezrasilvera did I convince you? If not, can you suggest a text that would satisfy you?

Comment (ezrasilvera) [reply tests/vmi_headless_test.go:121]: @dankenigsberg I just wanted to be sure that indeed you don't think it's needed.  Your warning and test is about a **diff** value, while the printouts are of the individual values. So basically the tester/user will calculate it to see what happen (unless one of the values is really an outlier). 
I thought of something like:
`memory difference between headless (%s) and normal (%s), is %sM  but should be roughly 16M`
If you have strong opinion against it, leave it as is and replay, it's not a big deal ..

Comment (dankenigsberg) [reply tests/vmi_headless_test.go]: I find this copious and annoying, but review in king.

**Comment by dhiller** [comment]
@dankenigsberg I figure there's something wrong with the data types regarding your `fmt.Sprintf`: https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/4991/pull-kubevirt-e2e-k8s-1.18/1364243306304770048

Have you tried whether this works locally?

**Comment by dankenigsberg** [comment]
> @dankenigsberg I figure there's something wrong with the data types regarding your `fmt.Sprintf`: https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/4991/pull-kubevirt-e2e-k8s-1.18/1364243306304770048
> 
> Have you tried whether this works locally?

No I have not. Sorry for the noise.

/hold

**Comment by dankenigsberg** [comment]
/hold cancel
(fixed to `%d` and tested with `make functest`)

**Review by dhiller — APPROVED** [review]
/approve

**Comment by dankenigsberg** [comment]
pull-kubevirt-e2e-k8s-1.17-rook-ceph failed with two unrelated timeouts:
```
[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system] VM Live Migration Starting a VirtualMachineInstance rook-ceph live migration cancelation should be able to cancel a migration [test_id:2731] with OCS Disk (using ISCSI IPv4 address) expand_less
[Serial]ImageUpload Upload an image and start a VMI with PVC on rook-ceph [test_id:4621] Should succeed DataVolume expand_more
```

**Pull request merged**